src/set.rs

```diff
@@ -452,8 +452,6 @@ impl<T, S> HashSet<T, S, Global> {
 
 impl<T, S, A> HashSet<T, S, A>
 where
-    T: Eq + Hash,
-    S: BuildHasher,
```

Comment:
Why is this changed?

Reply:
The bounds for `with_hasher` were removed. This removes them for `with_hasher_in` as well.